public/components/datasources/components/manage/accelerations/acceleration_table.tsx

```diff
@@ -168,6 +170,9 @@ export const AccelerationTable = ({
                 <CreateAccelerationFlyoutButton
                   dataSourceName={dataSourceName}
                   renderCreateAccelerationFlyout={renderCreateAccelerationFlyout}
+                  loadStatus={loadStatus}
+                  startLoading={startLoading}
+                  stopLoading={stopLoading}
```

Comment:
Why are we only passing, hooks to load table fields? Create acceleration flyout uses:
1. Load databases: https://github.com/opensearch-project/dashboards-observability/blob/ce9db203e4922d0b3e83e5ce6f09b1e4c9f23a66/public/components/datasources/components/manage/accelerations/create_accelerations_flyout/selectors/selector_helpers/load_databases.tsx#L41
2. Load tables/accelerations: https://github.com/opensearch-project/dashboards-observability/blob/ce9db203e4922d0b3e83e5ce6f09b1e4c9f23a66/public/components/datasources/components/manage/accelerations/create_accelerations_flyout/selectors/selector_helpers/load_objects.tsx#L48-L57
3. Load table Columns: which is edited here. 
4. Direct query Hook: https://github.com/opensearch-project/dashboards-observability/blob/ce9db203e4922d0b3e83e5ce6f09b1e4c9f23a66/public/components/datasources/components/manage/accelerations/create_accelerations_flyout/create/create_acceleration_button.tsx#L30
Should all of the above lifted up in state?

Reply:
Hmm, that makes sense. Then, I can make the change such that the [giant hook](https://github.com/opensearch-project/dashboards-observability/blob/main/public/components/datasources/components/manage/data_connection.tsx#L76-L83) containing all the hooks (I'll add `useDirectQuery` as well) gets passed in as a prop. I can also create a type for `cacheLoadingHooks`.

Reply:
Ideally, we should use `Redux` or `useContext` to pass down element deep in tree. If that takes time we can use the giant hook. For the Giant hook, we'll have to expose direct query in public/types to be re-used in workbench